integration-cli/docker_cli_network_unix_test.go

```diff
@@ -227,6 +318,27 @@ func (s *DockerNetworkSuite) TestDockerNetworkIpamMultipleNetworks(c *check.C) {
 	}
 }
 
+func (s *DockerNetworkSuite) TestDockerNetworkCustomIpam(c *check.C) {
+	// Create a bridge network using custom ipam driver
+	out, _, err := dockerCmdWithError("network", "create", "--ipam-driver", dummyIpamDriver, "br0")
```

Comment:
Confirmed, error thrown by `dockerCmdWithError` contains all the error info. Not sure how I missed it the first time. Thanks.